homeassistant/helpers/template/extensions/collection.py

```diff
@@ -84,6 +92,11 @@ def __init__(self, environment: TemplateEnvironment) -> None:
                     self.is_tuple,
                     as_test=True,
                 ),
+                TemplateFunction(
+                    "natural_sort",
+                    self.natural_sort,
+                    as_test=True,
```

Comment:
There is no need to register this separately. Instead, just set the as_test flag on the definition above.